.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.5
```

Comment:
What about keeping this in-sync with Foodsoft?
I'm ok with leaving it like this, though, and wait until we update Foodsoft's Ruby version.

Reply:
Upgrading from Ruby 2.6 to 2.7 seams like a big [step](https://github.blog/2020-08-25-upgrading-github-to-ruby-2-7/) with a lot of work. I can't estimate how much work it would be to downgrade sharedlists. Hence I'm in favor of leaving it at Ruby 2.7.

Reply:
Perhaps 2.6.9?
https://github.com/foodcoops/foodsoft/blob/master/.ruby-version

Reply:
From a practical point of view I would leave it at Ruby 2.7 because it's less work to adapt the code to Ruby 2.6.